src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue

```diff
@@ -8,115 +8,142 @@
 
 ```vue
 	<template>
-		<ul class="nav">
-			<NcAppNavigationCaption
-				name="Your caption goes here">
-				<template #actions>
-					<NcActionButton>
-						<template #icon>
-							<Plus :size="20" />
+		<div class="styleguide-nc-content">
```

Comment:
If we don't need to check, how focus-trap behaves, then mocking is fine. Otherwise I'd suggest to wrap it in NcContent, or we can't do it several times on the page?
![image](https://github.com/user-attachments/assets/40b5c080-105d-45f3-861d-0e473d9840f0)